modules/500-openvpn/openapi/config-values.yaml

```diff
@@ -221,10 +218,6 @@ properties:
     type: boolean
     default: false
     x-examples: [true, false]
-  storageClass:
```

Comment:
whats up with a storage class here?

Reply:
There was a migration for this https://github.com/deckhouse/deckhouse/blob/main/modules/500-openvpn/hooks/easyrsa_migration.go#L109-L111
Now there is a conversion 1->2 (https://github.com/deckhouse/deckhouse/pull/1729/files#diff-440419160488895ea321c1b73dfae715e1f97e2f35d891c0441ab31556e86f9c).